client/peerset/src/lib.rs

```diff
@@ -178,7 +178,7 @@ pub struct PeersetConfig {
 	/// Lists of nodes we should always be connected to.
 	///
 	/// > **Note**: Keep in mind that the networking has to know an address for these nodes,
-	/// >			otherwise it will not be able to connect to them.
+	/// >    otherwise it will not be able to connect to them.
```

Comment:
I don't understand why un-align these.

```suggestion
	/// >           otherwise it will not be able to connect to them.
```